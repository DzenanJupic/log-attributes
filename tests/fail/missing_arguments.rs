#[log_attributes::log()]
fn missing_arguments() {}

fn main() {}
