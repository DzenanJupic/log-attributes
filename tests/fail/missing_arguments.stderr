error: No arguments supplied
       help: log requires a log level and a fmt string
       example: #[log(info, "{fn} returned {return}")]
 --> tests/fail/missing_arguments.rs:1:1
  |
1 | #[log_attributes::log()]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `log_attributes::log` (in Nightly builds, run with -Z macro-backtrace for more info)
